src/G2/Equiv/InitRewrite.hs

```diff
@@ -19,7 +19,9 @@ initWithRHS s b r =
            }
       b' = b { input_names = map idName $ ru_bndrs r }
   in
-  markAndSweepPreserving emptyMemConfig s' b'
+  --markAndSweepPreserving emptyMemConfig s' b'
+  -- TODO temporary fix
+  (s', b')
```

Comment:
I don't think we have to do this before merging, but we should revisit this at some point.  I recall that it sped things up quite a bit?